JetUtilities/interface/HistogramUtilities.hh

```diff
@@ -91,6 +92,10 @@ namespace HistUtil{
     //      (i.e. if the AxisDirection=Y, then the metric will be plotted along X)
     vector<pair<double,double> > getHistogramMetric2D(HistogramMetric, TH2*, AxisDirection, const vector<string>&);
     vector<pair<double,double> > getHistogramMetric2D(string, TH2*, AxisDirection, const vector<string>&);
+
+    void adjust_fitrange(TH1* h,double& min,double& max);
+    int number_filled_bins(TH1* h, double min, double max);
+    TF1* fit_gaussian(TH1*& hrsp, const double nsigma, const double jtptmin, const int niter, const int verbose);
```

Comment:
If you're going to add fit_gaussian here why not also add fit_dscb?

Reply:
Because I am not sure what that is. And I have never got a chance to use this fit_dscb() yet.